src/app/movies/movies.component.html

```diff
@@ -20,7 +20,7 @@
   <button mat-raised-button>Usuń</button>
   <button mat-raised-button>Dodaj do ulubionych</button>
   <ng-template #movieView>
-    <p>{{movie.type}} {{movie.id}}</p>
+    <p>{{movie.type}} {{movie.id}} {{movie.statistics}} {{movie.createdAt}}</p>
```

Comment:
tu soibie wyświetlisz te viewCount itp

Reply:
a jesli decydujesz się na przechowywanie statistics, to będzie coś w stylu movie.statictics.viewCount